### DISCUSSION TITLE
Faster Method to Navigate to Records

### DISCUSSION THREAD

**Discussion by dd-manderson** [burkybang/SuiteAdvanced-Feedback#149] [Q&A] [answered]
Often when debugging, the only piece of information that I have is an internal ID.  For example, right now I am debugging an RA issue and I know that the internal ID is 12345.  My current method is to find an existing RA, open it, and then replace the internal ID in the Querystring.    I would like a quick way (keyboard shortcut?) to have a dialog popup with an input box, paste the internal ID and then navigate to the record.  Bonus points for automatically figuring out if it's a transaction, entity, item and crm record, but would totally fine with preselecting that first.

**Comment by burkybang** [comment, marked as answer]
This is already a feature! Use the global search. Enter the record type and internal ID separated by a colon or comma.

Examples:

```
salesorder:123
employee,456
```

You can also open multiple records like this:

```
customer:123,124,125
```

Object script IDs are also supported:

```
custsearch_my_search
```
